README.md

```diff
@@ -8,6 +8,21 @@ For details, see the executable documentation in the `vroom/` directory or the
 helpfiles in the `doc/` directory. The helpfiles are also available via
 `:help codefmt` if codefmt is installed (and helptags have been generated).
 
+# Supported File-types
```

Comment:
Consider how many different places we embed this full list and whether each one is worth keeping up to date. My hunch is "barely yes" for this one, but see if you can prune any other duplication when adding this. Some recent PRs had some noisy back-and-forth about getting all those updated and running vimdoc after, and sometimes I just give up and plan on resolving it all myself after the main PR is merged.

I'd suggest 2 or 3 complete lists are okay (the vimdoc and maybe 1 or 2 other places), and everywhere else having a short list of the most commonly used ones with an "…and many more (see `:help codefmt.txt`).